hikari/impl/rest.py

```diff
@@ -1256,6 +1278,7 @@ async def edit_message(
         content: undefined.UndefinedOr[typing.Any] = undefined.UNDEFINED,
         *,
         embed: undefined.UndefinedNoneOr[embeds_.Embed] = undefined.UNDEFINED,
+        embeds: undefined.UndefinedNoneOr[typing.Sequence[embeds_.Embed]] = undefined.UNDEFINED,
```

Comment:
Missing check for both `embed` and `embeds` as well for the check of passing a non-collection to `embeds`.

Reply:
Should have both imo. If not, people will start to see unexpected behavior. Also, would it be worth to move this logic to a super `_create_message` and `_edit_message` as it is now the same for both webhooks and bots?

Reply:
The nonce field isn't present on execute webhook and this isn't present yet but you can only provide flags on execute webhook, not create message

Reply:
Ah ok 👍

Reply:
Actually, we could just always send nonce as `None` in the webhook message, so then we can merge the logic

Reply:
Idk what you mean by that but nonce is a user defined value, also can't merge the edit message endpoints since webhooks can't edit flags

Reply:
Ik,  you can then default those depending on the call.

```py
async def _create_message(route = UNDEFINED, nonce = UNDEFINED, flag = UNDEFINED, avatar = UNDEFINED, username = UNDEFINED, ***):
    ...
    
async def create_message(nonce):
    await _create_message(nonce)
    
async def execute_webhook(flag, avatar, username):
    await _create_message(flag, avatar, username)
```

Reply:
I've already merged them but didn't use that style cause there's a better way to handle it in this case